r/R/data.R

```diff
@@ -6,6 +6,214 @@
 # option. This file may not be copied, modified, or distributed
 # except according to those terms.
 
+#' Create a small demo matrix and fragment object.
+#' 
+#' Downloads a 10x Genomics dataset consisting of 3k cells, then performs optional QC and subsetting.  Holds subsetted objects in disk,
+#' and returns a list with both the matrix and fragments. 
+#' @param directory (character) Where the input/output data should be stored.  If NULL, a temporary directory is created.
+#' @param filter_qc (bool) Whether to filter both the RNA and ATAC data using QC information.
+#' @param subset (bool) Whether to subset to only genes/insertions on chromosome 4 and 11.
+#' @param timeout (numeric) Timeout for downloading files in seconds.
+#' @param remove_input_data (logical) Whether to remove the downloaded non-procesed matrix, frags, gencode transcripts, and gencode genes
```

Comment:
Spell-checking plugin to the rescue (though mine still has tons of false positives so I'll probably leave it off most of the time)
```suggestion
#' @param remove_input_data (logical) Whether to remove the downloaded non-processed matrix, frags, gencode transcripts, and gencode genes
```